howto/application/resources.md

```diff
@@ -1,23 +1,27 @@
-Anbox Cloud provides a set of [instance types](https://discourse.ubuntu.com/t/instance-types-reference/17764) that define the resources available to a container. For example, if you start a container for an application that uses the instance type `a4.3`, the container is assigned 4 vCPU cores, 3 GB of RAM and 3 GB of disk space.
+Anbox Cloud provides a set of instance types that define the resources available to a container. Often your application may require resources that do not correspond to any of the provided instance types. In such cases, you can override some or all of the resource requirements in your application manifest.
 
-If your application requires resources that do not correspond to any of the provided instance types, you can override some or all of the resource requirements in your [application manifest](https://discourse.ubuntu.com/t/application-manifest/24197).
+Define the required `cpus`, `memory`, `disk size` values for the `resources` attribute in the application manifest. 
 
-For example, if you are using the instance type `a4.3` but require 5 GB of disk space, use the following application manifest:
+For example, the following application manifest results in allotting 6 vCPU cores, 4 GB memory and 8 GB disk size instead of the standard 4 vCPU cores, 3 GB memory and 3 GB disk size for an `a4.3` instance type.
 
-```
+```yaml
 name: my-application
 instance-type: a4.3
 resources:
-  disk-size: 5GB
+  cpus: 6
+  memory: 4GB
+  disk-size: 8GB
```

Comment:
The new example makes less sense to me in practice since if people customize each resource, the `instance-type` can be omitted here in the manifest file. If we really want to demonstrate how to override one or two certain resources that were given from `instance-type` with the `resources` attribute. I'd suggest dropping either `cpus` or `memory` in the above example.

Reply:
Makes sense, modified as you suggested